data/applications/nvidia.yaml

```diff
@@ -0,0 +1,17 @@
+metadata:
+  name: nvidia
+  annotations:
+    opendatahub.io/categories: 'GPU'
+spec:
```

Comment:
csvName should be needed like others.
Looks the csvName for nvidia gpu operator is `gpu-operator-certified`

Reply:
Added

Reply:
@Xaenalt I believe you have a cluster that installed the addon. I think we should check the addon operator name of NVIDIA addon. could you check it?

Reply:
```
oc get csv -n redhat-nvidia-gpu-addon
NAME                                               DISPLAY                           VERSION           REPLACES                                           PHASE
configure-alertmanager-operator.v0.1.436-6b176b2   configure-alertmanager-operator   0.1.436-6b176b2   configure-alertmanager-operator.v0.1.434-db3fd6e   Succeeded
gpu-operator-certified.v1.10.1                     NVIDIA GPU Operator               1.10.1            gpu-operator-certified.v1.10.0                     Succeeded
nfd.v4.10.0                                        Node Feature Discovery Operator   4.10.0                                                               Succeeded
nvidia-gpu-addon-operator.1.0.0                    NVIDIA GPU Add-on                 1.0.0                                                                Succeeded
ose-prometheus-operator.4.10.0                     Prometheus Operator               4.10.0                                                               Succeeded
rhods-operator.1.12.0-4                            Red Hat OpenShift Data Science    1.12.0-4          rhods-operator.1.11.0-5                            Succeeded
route-monitor-operator.v0.1.420-b65f47e            Route Monitor Operator            0.1.420-b65f47e   route-monitor-operator.v0.1.418-6459408            Succeeded
```

Reply:
I think we'd probably want to use gpu-operator-certified since that will work for operator or addon by the looks of it

Reply:
@dlabaj what do you think about this? should we use `nvidia-gpu-addon-operator` or `gpu-operator-certified` ? To me, the first name should be the right because this doc is about addon .. not certified operator.

Reply:
hi, sorry to re-open this, but are you sure the csvName should be `gpu-operator-certified`? RH is supporting the addon @VedantMahabaleshwarkar

Reply:
AFAIK we (RH) support only the add-on, not the operator. +1 to @Jooho